src/components/Tabs/Tabs.scss

```diff
@@ -318,6 +319,14 @@ $focus-state-box-shadow-color: rgba(0, 0, 0, 0.8);
 
   .Tab {
     padding: spacing(tight) spacing(extra-tight);
+
+    &:hover,
+    &:active,
+    &:focus {
```

Comment:
Why the specificity here? This should just work directly under `.Tab` unless we override the font weights on focus, active, and hover?

Reply:
There is